client/src/components/AppointmentCreateForm.js

```diff
@@ -53,11 +46,11 @@ const AppointmentCreateForm: () => React$Node = () => {
     required: "required"
   };
   
-  var startTime = new Date(newFromTime);
-  var endTime = new Date(newToTime);
+  var startTime = () => newFromTime.format('X');
```

Comment:
why are these functions? right now it makes the line `startTime()>=endTime()` looks awkward because they should not be functions.